examples/selfreferential.py

```diff
@@ -0,0 +1,25 @@
+import os
+import re
+from pathlib import Path
+
+from wrdcld import make_word_cloud
+
+FOLDER_PATH = Path("wrdcld")
+
+all_text = ""
+for filename in os.listdir(FOLDER_PATH):
+    if filename.endswith(".py"):
+        with open(FOLDER_PATH / filename, "r", encoding="utf-8") as f:
+            all_text += f.read()
+
+cleaned_text = re.sub(r"[^a-zA-Z0-9_\s]", " ", all_text)
```

Comment:
do you think we should add some cleaning utilities and common word removal stuff? Or does it make more sense to leave that to the user? do one thing and do it well kind of mentality

Reply:
> do one thing and do it well kind of mentality

I would vote for this, especially early on

Reply:
It might make sense to assemble some good routines for the examples and the docs, if nothing else.